### PR TITLE
ter

### PR DESCRIPTION
**Pull request by lavahotsnacks**

### CODE REVIEW

File: firebase.json [hunk 1]
```diff
@@ -20,9 +20,7 @@
     "firestore": {
       "port": 8080
     },
-    "hosting": {
-      "port": 5000
-    },
+
```
Comment (lavahotsnacks) [inline firebase.json]:
```suggestion
    "hosting": { 
         "port: 5000
     },
```

**Comment by lavahotsnacks** [comment]
IP pa to dont approve

File: firebase.json
Comment (29-29) [inline firebase.json:1]: changed what?

**Review by 29-29 — COMMENTED** [review]
would like to know what relevant changes were made

**Comment by 29-29** [comment]
> IP pa to dont approve

You can make this pull request into a draft if it's in progress.

File: firebase.json [hunk 2]
```diff
@@ -20,9 +20,9 @@
     "firestore": {
       "port": 8080
     },
-    "hosting": {
-      "port": 5000
-    },
+    "hosting": { 
+         "port": 5000
```
Comment (29-29) [inline firebase.json:24]: pointless modification lol

**Comment by lavahotsnacks** [comment]
sorry for the late reply but it didn't notify my phone, as i only saw this notification through my laptop's mail app.

**i did _not_ make this pull request or write those comments**, that was during the time when i asked sir Lucman to teach me how to use Github and showed examples (cloning, difference of fork and branching, removing the code line, then putting it back, the process, etc)

**Comment by 29-29** [comment]
Understood.

**Pull request closed without merging**